internal/ips/request.go

```diff
@@ -86,10 +119,14 @@ func (c *Client) Request() *cobra.Command {
 	requestIPCmd.Flags().StringVarP(&metro, "metro", "m", "", "Code of the metro where the IP Reservation will be created")
 	requestIPCmd.Flags().IntVarP(&quantity, "quantity", "q", 0, "Number of IP addresses to reserve.")
 	requestIPCmd.Flags().StringSliceVar(&tags, "tags", nil, "Tag or Tags to add to the reservation, in a comma-separated list.")
+	requestIPCmd.Flags().IntVarP(&cidr, "cidr", "i", 0, "The size of the desired subnet in bits.")
```

Comment:
we don't need to define a short argument. Let's use `IntVar` instead of `IntVarP` and remove the `"i"`. I don't think it is intuitive and users can just use `--cidr`.